modules/core/shared/src/main/scala/lucuma/core/model/AngularSize.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.Show
+import lucuma.core.math.Angle
+import monocle.Focus
+import monocle.Lens
+
+/**
+ * Angular size of an object in the sky. The major axis is the longest one, but we don't enforce
+ * this.
+ */
+final case class AngularSize(majorAxis: Angle, minorAxis: Angle)
```

Comment:
There is this `https://github.com/gemini-hlsw/react-aladin/blob/master/demo/src/main/scala/react/aladin/package.scala#L22` though it is more basic

Reply:
Never mind, it should be the other way around. Make `react-aladin` depend on `AngularSize`

Reply:
I'm not sure both classes have the same definition. In `Fov` the angles are aligned to the `x` and `y` axes, while in `AngleSize` they are not: it describes the ellipse that inscribes the target.